spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/RdbmsStreamDefinitionRepository.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+
+import javax.sql.DataSource;
+
+import org.springframework.cloud.dataflow.core.StreamDefinition;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.util.Assert;
+
+/**
+ * RDBMS implementation of {@link StreamDefinitionRepository}.
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+public class RdbmsStreamDefinitionRepository extends AbstractRdbmsKeyValueRepository<StreamDefinition> implements StreamDefinitionRepository {
+
+	public RdbmsStreamDefinitionRepository(DataSource dataSource) {
```

Comment:
probably want to have the user to pass in the prefix and suffix.  As they may already have tables with these names.  And if they don't use the defaults.